torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -51,6 +51,18 @@ Operation noop(const Node* n) {
   return [](Stack& stack) { return 0; };
 }
 
+template<class T> bool is_equal(const c10::ListPtr<T>& lhs, const c10::ListPtr<T>& rhs) {
```

Comment:
yuck -- 140 lines of add stuff in our prim ops just to wrap up a vector? This is a huge code smell and is suggestive of what our users will have to do to use the List type as it is.

Reply:
it's not adding 140 lines. It's modifying 57 lines and adding 26.

Reply:
After my latest push, it's now only 49 modified and 26 added lines.